Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [root]
 name = "tikv"
-version = "0.0.1"
+version = "0.9.0-pre-ga"
```

Comment:
🤣 funny versioning scheme, makes software release life cycle a joke.
Better give it up this time and set it up at/after 1.0.0, i.e. GA